README.md

```diff
@@ -1,222 +1,74 @@
 # Stencil Utilities
 
-Customizable, composable and testable CSS utilities for mobile browsers.
+CSS utilities for mobile browsers.
```

Comment:
True, but starting with something more descriptive is a good idea. This line, plus the sentence below could be a lot clearer. [This](https://github.com/suitcss/suit/blob/master/doc/utilities.md) is probably the best explanation out there.